src/mavo.js

```diff
@@ -433,25 +434,29 @@ let _ = self.Mavo = $.Class(class Mavo {
 		}
 
 		if (!backend) {
-			backend = _.Functions.url(`${this.id}-${role}`)
-			          || _.Functions.url(`${this.id.toLowerCase()}-${role}`)
-			          || this.element.getAttribute(attribute) || null;
+			backend = _.Functions.url(`${this.id}-${role}`);
 		}
 
-		if (backend) {
-			backend = backend.trim();
+		if (!backend) {
+			backend = this.element.getAttribute(attribute) || null;
```

Comment:
I'd move `const attribute = "mv-" + role;` in here if it's not used anywhere outside this.

Reply:
Right. Done. Thanks!